simulation_ws/src/person_detection_simulation/config/kvs_config.yaml

```diff
@@ -0,0 +1,44 @@
+kinesis_video_streamer:
+  ros__parameters:
+    aws_client_configuration:
+      # Specifies where you want the client to communicate. Examples include us-east-1 or us-west-2.
+      region: "us-east-1"
```

Comment:
`cloudwatch_logs_config.yaml` and `cloudwatch_metrics_config.yaml` show `us-west-2`. Should change this to `us-west-2` to be consistent.